databa.py

```diff
@@ -0,0 +1,50 @@
+import peewee
+from peewee import *
+
+db = SqliteDatabase('news_bot.db')
+
+
+class Stat(Model):
+    name = CharField(unique=True)
+    business_number_of_requests = IntegerField()
+    entertainment_number_of_requests = IntegerField()
+
+
+    class Meta:
+        database = db
+        order_by = 'name'
+        db_table = 'stats'
+
```

Comment:
Here it's better to add one more blank line (PyCharm may help you to avoid such problems)